### ISSUE TITLE
2.0 features .NET core

### ISSUE THREAD

**Issue by mdissel** [rivantsov/vita#63]
The readme page is mentioning that features are missing in the new 2.0 version. Is that still valid?

https://github.com/rivantsov/vita/blob/master/README.md

**Comment by rivantsov** [comment]
yes, still missing. Query cache and data cache should be coming soon

**Comment by Michaellok1992** [comment]
Hi @rivantsov , may i know what is not yet to be port over from previous version? and also what feature is coming soon?

**Comment by rivantsov** [comment]
Hi
Not ported yet are: 
1.  Pure ORM features: query caching, data cache, Include options for queries authorization framework
2. Logging modules, and some other extra modules 
3. Web Api stack integration - you can still build Web apps with VITA, just without some extra convenience features, like automatic logging 
PS some stuff is coming tonight
